configure.ac

```diff
@@ -9806,6 +9806,16 @@ then
     fi
 fi
 
+AC_ARG_ENABLE([old-extdata-fmt],
```

Comment:
Please add comment for when this old option would be needed. Add that it only applies to compatibility layer and certain API's.